games-action/dave_gnukem/dave_gnukem-1.0.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="2D platform shooter inspired by Duke Nukem 1"
+DESCRIPTION="Dake Gnukem is a retro-style 2D scrolling platform shooter \
+similar to, and inspired by, Duke Nukem 1.
+
+The year is \$CURRENTYEAR+8. 
+An evil genius, Dr Proetton, has been hired by the CIA \
+to infect the world's computers with a virus called SystemD, crippling them. \
+Only you can stop him. You must find the floppy disk with the \
+Devuan Antivirus on it, and install it on the master computer, \
+which is hidden in Vault7.
+TL;DR: Shoot anything that moves!
+
+Controls:
+Left/right -> Move left/right
+Ctrl -> Jump
+Alt -> Shoot
+Up -> Action key (open doors, use teleporters, etc.)
+Escape -> Open the menu
+"
+
+HOMEPAGE="http://djoffe.com/gnukem/"
+REVISION="1"
+COPYRIGHT="1998-2019 David Joffe"
+LICENSE="GNU GPL v2"
```

Comment:
From the README:
> The source code itself (and most of the data/sprites) for Dave Gnukem are dual-licensed under MIT license or GPL.
So shouldn't both be listed here? Not sure since it's not an logical and but rather "or"…

Reply:
I'd list both.  Does this recipe include the data files, or do we need another recipe to grab and install the data files for the game to work?

Reply:
Maybe "CC BY 3.0" should be added also: https://github.com/davidjoffe/dave_gnukem#license-and-additional-credits

Reply:
@scottmc This recipe includes the data files. 
I'll include all the three licenses suggested here.